pkg/util/version/upgrades.go

```diff
@@ -0,0 +1,27 @@
+package version
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type Upgrade struct {
```

Comment:
+1 on the concept in this file.  Some doubts about naming though.  "Upgrade" is really some sort of parallel version stream.  "Latest" means "this is what we install"?  Perhaps Streams and Install might be better names?